app/controllers/users_controller.rb

```diff
@@ -143,7 +146,7 @@ def should_include_permissions?
 
   def paginate_users
     if any_filter?
-      unless @users.kind_of?(Array)
+      if not @users.is_a?(Array)
```

Comment:
Actually it would be better to reverse the conditional here.  The rubocop rule is to avoid `unless ... else` constructs.  Using `if not` will fix the warning, but not the intent behind the warning.